carp/pytorch/tests/test_dataset.py

```diff
@@ -1,7 +1,8 @@
 import pytest
```

Comment:
Do we need both `carp/pytorch/tests/test_dataset.py` and `carp/pytorch/test_dataset.py`?